cli/internal/cmd/upgradeapply.go

```diff
@@ -157,10 +158,10 @@ func (u *upgradeApplyCmd) migrateTerraform(cmd *cobra.Command, file file.Handler
 	u.log.Debugf("Using Terraform variables:\n%v", vars)
 
 	opts := upgrade.TerraformUpgradeOptions{
-		LogLevel:   flags.terraformLogLevel,
-		CSP:        conf.GetProvider(),
-		Vars:       vars,
-		Targets:    targets,
+		LogLevel: flags.terraformLogLevel,
+		CSP:      conf.GetProvider(),
+		Vars:     vars,
+		// Targets:    targets,
```

Comment:
This is also a left-over from debugging, right?

Reply:
Just that we are on the same page here, the current code provides no solution which should be merged into main, as it disables the Terraform migration for all CSPs, right? 

Since I'm reading the code for the first time, maybe @malt3 @derpsteb, or @msanft can chime in on how to best handle the migration part here. My question would be, why this did not come up during the GCP version of this PR.

Reply:
yes the current commit cannot be merged. I also wonder why it didn't cause problems in GCP

Reply:
I believe this was an oversight on my side. I was pretty sure that this worked in my tests but looking at the code, we will have to either drop targets altogether or include the full list of moved resources in the list of targets.

Reply:
From what I see is that GCP should work correctly, since it does not migrate any resources. But as soon as we want to migrate a resource Terraform needs to migrate _ALL_ moved resources. 
> Resource instances in your current state have moved to new addresses in the
latest configuration. Terraform must include those resource instances while
planning in order to ensure a correct result, but your -target=... options to
not fully cover all of those resource instances.

Reply:
We decided to remove targets in AB#3238